models.py

```diff
@@ -1,22 +1,12 @@
-from keras.models import *
-from keras.layers import *
-from keras import layers
-from keras.regularizers import l2
+from tensorflow.keras.models import Model
+from tensorflow.keras.layers import Conv2D, Concatenate, ZeroPadding2D, BatchNormalization, Activation, MaxPooling2D, UpSampling2D, Input, Layer
+from tensorflow.keras import layers
+from tensorflow.keras.regularizers import l2
+import tensorflow as tf
 
 resnet50_Weights_path='./pretrained_model/resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5'
-IMAGE_ORDERING ='channels_last'
-MERGE_AXIS=-1
 
-
-def one_side_pad( x ):
-    x = ZeroPadding2D((1, 1), data_format=IMAGE_ORDERING)(x)
-    if IMAGE_ORDERING == 'channels_first':
-        x = Lambda(lambda x : x[: , : , :-1 , :-1 ] )(x)
-    elif IMAGE_ORDERING == 'channels_last':
-        x = Lambda(lambda x : x[: , :-1 , :-1 , :  ] )(x)
-    return x
-
-def identity_block(input_tensor, kernel_size, filters, stage, block):
+def identity_block(input_tensor, kernel_size, filters, stage, block, data_format='channels_last'):
```

Comment:
Added `data_format` as a parameter everywhere, rather than the global constant `IMAGE_FORMAT`. Also used the same name as `tf` does for transparency.